indico/modules/events/sessions/controllers/management/sessions.py

```diff
@@ -185,6 +186,16 @@ def _get_defaults(self):
                 'acl': acl}
 
 
+class RHSessionACL(RHManageSessionBase):
```

Comment:
Same code as RHContributionACL except for `self.session` and the base class? in that case a util function would be better so you can just do `return render_acl(self.session)`